### PR TITLE
Inline versions of upgraded formulae

### PR DESCRIPTION
**Pull request by dasmfm**
- [+ ] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [+] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [+] Have you added an explanation of what your changes do and why you'd like us to include them?
- [-] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/test/PATH_spec.rb).
- [+] Have you successfully run `brew style` with your changes locally?
- [+] Have you successfully run `brew tests` with your changes locally?

-----
  Sometimes, when there are multiple upgraded formulae with large output, I'm confused which version will I receive after upgrade. This small patch helps to understand that just when upgrading time comes.

### CODE REVIEW

File: Library/Homebrew/cmd/upgrade.rb [hunk 1]
```diff
@@ -185,7 +185,12 @@ def upgrade_formula(f)
     end
     fi.prelude
 
-    oh1 "Upgrading #{Formatter.identifier(f.full_specified_name)} #{fi.options.to_a.join " "}"
+    if f.optlinked?
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cmd/upgrade.rb]:
```suggestion
    upgrade_version = if f.optlinked?
```

File: Library/Homebrew/cmd/upgrade.rb [hunk 2]
```diff
@@ -185,7 +185,12 @@ def upgrade_formula(f)
...
+      oh1 "Upgrading #{Formatter.identifier(f.full_specified_name)} #{Keg.new(f.opt_prefix).version} ->"\
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cmd/upgrade.rb]:
```suggestion
      "#{Keg.new(f.opt_prefix).version} -> #{f.pkg_version}"
```

File: Library/Homebrew/cmd/upgrade.rb [hunk 3]
```diff
@@ -185,7 +185,12 @@ def upgrade_formula(f)
...
+      " #{f.pkg_version} #{fi.options.to_a.join " "}"
+    else
+      oh1 "Upgrading #{Formatter.identifier(f.full_specified_name)} -> #{f.pkg_version} #{fi.options.to_a.join " "}"
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cmd/upgrade.rb]:
```suggestion
      "-> #{f.pkg_version}"
```

File: Library/Homebrew/cmd/upgrade.rb [hunk 4]
```diff
@@ -185,7 +185,12 @@ def upgrade_formula(f)
...
+    end
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cmd/upgrade.rb]:
```suggestion
    end
    oh1 "Upgrading #{Formatter.identifier(f.full_specified_name)} #{upgrade_version} #{fi.options.to_a.join(" ")}"
```

**Comment by dasmfm** [comment]
@MikeMcQuaid, thanks. I'm not a Ruby developer, so just accept everything :)

**Comment by dasmfm** [comment]
@MikeMcQuaid , looks good from the code, and I'm currently out of outdated apps to test.

File: Library/Homebrew/cmd/upgrade.rb [hunk 5]
```diff
@@ -185,7 +185,13 @@ def upgrade_formula(f)
     end
     fi.prelude
 
-    oh1 "Upgrading #{Formatter.identifier(f.full_specified_name)} #{fi.options.to_a.join " "}"
+    upgrade_version = if f.optlinked?
+      "#{Keg.new(f.opt_prefix).version} -> #{f.pkg_version}"
+      " #{f.pkg_version} #{fi.options.to_a.join " "}"
```
Comment (Bo98) [inline Library/Homebrew/cmd/upgrade.rb]:
```suggestion
```

**Comment by dasmfm** [comment]
Thanks @Bo98, now it's working as expected:

> ==> Upgrading go 1.13.8 -> 1.14

**Pull request merged**